profiles/serializers.py

```diff
@@ -78,3 +86,10 @@ def validate(self, attrs):
         if company.person == user:
             raise serializers.ValidationError({"error": "You can not view your company."})
         return attrs
+
+class FiltersQueryParamSerializer(serializers.Serializer):
+    filters = serializers.CharField(required=True)
+
+    def to_representation(self, instance):
+        representation = {"instance": "instance"}
+        return representation
```

Comment:
I have strong feeling that something is very wrong here. 

Serializers have nothing to do with query parameters.